### ISSUE TITLE
error in allure-docx setup command: 'install_requires' must be a string or...

### ISSUE THREAD

**Issue by altendky** [typhoon-hil/allure-docx#19]
```
altendky@lt:~/test-software$ venv/bin/pip install ./allure-docx
Processing ./allure-docx
    Complete output from command python setup.py egg_info:
    error in allure-docx setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Invalid requirement, parse error at "'+https:/'"
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-req-build-1kaoeesg/
You are using pip version 18.1, however version 19.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

While you can do `git+https://...` in a file that you `pip install -r the_file`, it isn't valid in `install_requires`.  You need to put the actual distribution package name into `install_requires` (`pyinstaller` in this case).  Anyways, I'm going to take a guess that `allure-docx` doesn't actually require pyinstaller, even if it is used during development.

If you want me to propose an alternate setup I could think it through and suggest something.

**Comment by Sup3rGeo** [comment]
I believe I needed the latest pyinstaller from source, that's why the `requirements.txt` has the git address, but I did not notice this issue you are presenting.

I think you are right, we don't need pyinstaller unless we are building the executable.

Yes please feel free to suggest any improvements!

**Comment by altendky** [comment]
Yeah, I've spent most of my time with my projects using an unreleased PyInstaller version as well.  :|  Unless you don't install allure-docx I'm not sure why you wouldn't get the error.  It may have changed with pip/setuptools versions but I'm not aware of that offhand.  Anyways, PR on the way.
